app/scripts/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -45,6 +46,12 @@ define(function (require, exports, module) {
       this.closePanel();
     },
 
+    _closePanelReturnToClients: function () {
+      this.navigate('settings/clients');
+      this.closePanel();
+      $.modal.close();
```

Comment:
Is this line needed? In [settings.js->_onNavigateFromChildView](https://github.com/mozilla/fxa-content-server/blob/c3cc9d4a462e9c96cd702a5e0161065c4d8c0139/app/scripts/views/settings.js#L137:L139) if the next panel does not need a modal, it should close the modal for you.